src/components/productdisplayitem.main.less

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+@import './../style/common.less';
+
+.itemdetail-component {
+  padding-top: 30px;
+
+  .itemdetail-assets {
+    width: 250px;
+    vertical-align: top;
+    display: inline-block;
+
+    .itemdetail-asset-container {
+      img {
+        width: 250px;
+        height: 250px;
+        object-fit: cover;
+      }
+    }
+  }
+
+  .itemdetail-details {
+    vertical-align: top;
+    padding: 30px;
+    display: inline-block;
+    width: 500px;
+  }
+
+  .itemdetail-addtocart {
```

Comment:
Add to cart button does not have styling. Perhaps add style for one of btn-round btn btn-primary btn-itemdetail-addtocart. Unless we want to wait for mock ups since these may change?

Reply:
Yes we want to wait, product page is being completely redesigned.